collect_app/src/test/java/org/odk/collect/android/forms/FormsRepositoryTest.java

```diff
@@ -125,12 +145,18 @@ public void save_addsId() {
     @Test
     public void save_addsHashBasedOnFormFile() {
         FormsRepository formsRepository = buildSubject();
-        Form form = buildForm(1L, "id", "version", getFormFilesPath()).build();
+        Form form = buildForm("id", "version", getFormFilesPath()).build();
         assertThat(form.getMD5Hash(), equalTo(null));
 
         formsRepository.save(form);
 
         String expectedHash = FileUtils.getMd5Hash(new File(form.getFormFilePath()));
         assertThat(formsRepository.get(1L).getMD5Hash(), equalTo(expectedHash));
     }
+
+    public Clock getMockClock() {
+        Clock mockClock = mock(Clock.class);
+        when(mockClock.getCurrentTime()).thenReturn(2L, 3L, 1L);
```

Comment:
Such a minor tweak but I do think it'd be nicer if the mock values were set in the tests themselves. For instance, it's hard when reading `getLatestByFormIdAndVersion_whenMultipleExist_returnsLatest` to see which one is the latest.

Reply:
Agreed. This was an artifact of initially using `@Before`. Being able to put this initialization inline is another nice side effect of only using the mock clock where needed.